### PR TITLE
Upgrade to Webpack v2

### PR DESCRIPTION
**Pull request by peter-mouland**

### CODE REVIEW

File: src/app/lib/webpack/createWebpackConfig.js [hunk 1]
```diff
@@ -51,26 +55,28 @@ module.exports = (deps) => {
...
-          'ios_saf >= 8',
-          'ie_mob 11',
-          'Android >= 4'
-        ],
-        cascade: false,
-        add: true,
-        remove: true
-      }
-      )]
+        new ExtractTextPlugin({ filename: '[name].css', disable: false, allChunks: true }),
```
Comment (rogerfranklin2) [inline src/app/lib/webpack/createWebpackConfig.js]: why `disable:false` what does this do

File: src/app/lib/webpack/createWebpackConfig.js [hunk 2]
```diff
@@ -18,30 +18,34 @@ module.exports = (deps) => {
...
           },
           {
             test: /\.scss$/,
             exclude: /node_modules/,
-            loader: ExtractTextPlugin.extract('style', [
-              `css?-autoprefixer&sourceMap${minify ? '&minimize' : ''}`,
-              'postcss',
-              'sass?outputStyle=expanded'].join('!'))
+            loader: ExtractTextPlugin.extract({
+              fallbackLoader: 'style-loader',
```
Comment (rogerfranklin2) [inline src/app/lib/webpack/createWebpackConfig.js]: do any of the loaders need a `fallbackLoader`

File: src/app/lib/webpack/createWebpackConfig.js (hunk 1 above)
Comment (RichMatthews) [inline src/app/lib/webpack/createWebpackConfig.js]: good spot @rogerfranklin2. we will remove.

File: src/app/lib/webpack/createWebpackConfig.js (hunk 2 above)
Comment (peter-mouland) [inline src/app/lib/webpack/createWebpackConfig.js]: nope, just that one

**Pull request merged**